uber/templates/registration/index_base.html

```diff
@@ -124,9 +124,11 @@
         <tbody>
         {% for attendee in attendees %}
             <tr {% if attendee.badge_status == c.INVALID_STATUS %}bgcolor="#FFE4E1"{% endif %}>
-                {% block tablerows %}
+                {% block tablerows scoped %}
```

Comment:
I looked this up real quick since it's new so I didn't recognize it. For the benefit of others going through this, what `scoped` does here is make the `attendee` variable available to this block.